src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -28,41 +34,86 @@
 
 class GenomeStatus:
 
-    def __init__(self, original_id, resolved_id, attempted=False, filename=""):
+    def __init__(self,
+                 original_id,
+                 resolved_refseq_id,
+                 attempted=False,
+                 filename=""):
         self.original_id = ';'.join(original_id.split(','))
-        self.resolved_id = None if resolved_id == 'None' else resolved_id
-        self.attempted = True if attempted == 'True' else False
+        self.resolved_refseq_id = None if resolved_refseq_id == 'None' else resolved_refseq_id
```

Comment:
```suggestion
        self.resolved_refseq_id = "" if resolved_refseq_id == 'None' else resolved_refseq_id
```
It looks better to use empty string than None since the type of `resolved_refseq_id` will be only `str`. There are many type warnings if using None.

Please also update the places where it is evaluated like `if genome_obj.resolved_refseq_id is None:`.